### PR TITLE
initial work on views

### PR DESCRIPTION
**Pull request by nchurchill**
Created a header that is consistent across all views. Formatted a few other little things throughout the website.

### CODE REVIEW

File: src/Athena/Content/js/jquery.collapsible.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+/*!
+* Collapsible.js v1.2.0
+* https://github.com/jordnkr/collapsible
```
Comment (nlowe) [inline src/Athena/Content/js/jquery.collapsible.js]: Bring this in as an NPM package instead of including the source:

```bash
npm install --save collapsible.js
```

File: src/Athena/Content/js/site.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+﻿const jquery = require("jquery");
+window.$ = window.jQuery = jquery;
+
```
Comment (nlowe) [inline src/Athena/Content/js/site.js]: You will need to `require('collapsible.js`);` here if you want it included in the bundle

File: src/Athena/Controllers/HomeController.cs [hunk 3]
```diff
@@ -4,28 +4,43 @@
...
-        public IActionResult About()
+        [HttpGet("Login_Page")]
+        public IActionResult Login_Page()
         {
-            ViewData["Message"] = "Your application description page.";
+            return View();
+        }
+
+        [HttpGet("Add_Classes")]
+        public IActionResult Add_Classes()
```
Comment (nlowe) [inline src/Athena/Controllers/HomeController.cs]: This will need moved to a separate controller eventually. I am OK with leaving this here for now if you want to ease view development but it's something to keep in mind.

File: src/Athena/Controllers/HomeController.cs [hunk 4]
```diff
@@ -4,28 +4,43 @@
...
+        {
+            return View();
+        }
 
+        [HttpGet("Schedule_Page")]
+        public IActionResult Schedule_Page()
```
Comment (nlowe) [inline src/Athena/Controllers/HomeController.cs]: This will need moved to a separate controller eventually. I am OK with leaving this here for now if you want to ease view development but it's something to keep in mind.

File: src/Athena/Controllers/HomeController.cs [hunk 5]
```diff
@@ -4,28 +4,43 @@
...
+        {
             return View();
         }
 
-        [HttpGet("Contact")]
-        public IActionResult Contact()
+        [HttpGet("NewAccountPage")]
+        public IActionResult NewAccountPage()
```
Comment (nlowe) [inline src/Athena/Controllers/HomeController.cs]: This will need moved to a separate controller eventually. I am OK with leaving this here for now if you want to ease view development but it's something to keep in mind.

File: src/Athena/Controllers/HomeController.cs [hunk 6]
```diff
@@ -4,28 +4,43 @@
...
         {
-            ViewData["Message"] = "Your contact page.";
+            return View();
+        }
 
+        [HttpGet("Main_Page")]
+        public IActionResult Main_Page()
```
Comment (nlowe) [inline src/Athena/Controllers/HomeController.cs]: What's the difference between `Home` and `Main_Page`?

File: src/Athena/Views/Home/Add_Classes.cshtml [hunk 7]
```diff
@@ -0,0 +1,21 @@
+﻿@{
+    ViewData["Title"] = "Add Classes";
+}
+
+<p>Use this area to provide additional information.</p>
+
+<!--http://jordnkr.github.io/collapsible/-->
```
Comment (nlowe) [inline src/Athena/Views/Home/Add_Classes.cshtml]: What is the purpose of this comment?

File: src/Athena/Views/Home/Main_Page.cshtml [hunk 8]
```diff
@@ -0,0 +1,30 @@
+﻿@{
+    ViewData["Title"] = "Main_Page";
+}
+<br /><br /><br /><br /><br />
+
+<img src="~/images/UT-color-vert-trans.gif" class="responsive-img left" style="height:10em; max-width:100%" />
+<br /><br /><br /><br /><br /><br /><br /><br /><br /><br />
```
Comment (nlowe) [inline src/Athena/Views/Home/Main_Page.cshtml]: What is the purpose of these `<br/>`'s? Use padding or margins if you're trying to force spacing.

File: src/Athena/Views/Shared/_Layout.cshtml [hunk 9]
```diff
@@ -2,70 +2,111 @@
...
-        <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.7/css/bootstrap.min.css"
-              asp-fallback-href="~/lib/bootstrap/dist/css/bootstrap.min.css"
-              asp-fallback-test-class="sr-only" asp-fallback-test-property="position" asp-fallback-test-value="absolute" />
-        <link rel="stylesheet" href="~/css/site.min.css" asp-append-version="true" />
-    </environment>
+    <link rel="stylesheet" type="text/css" href="~/athena.css" />
+    <!--Let browser know website is optimized for mobile-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (nlowe) [inline src/Athena/Views/Shared/_Layout.cshtml]: Material Icons should be included by the bundler as a part of `materialize-css`. Is that not the case?

File: src/Athena/Views/Shared/_Layout.cshtml [hunk 10]
```diff
@@ -2,70 +2,111 @@
...
-        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-2.2.0.min.js"
-                asp-fallback-src="~/lib/jquery/dist/jquery.min.js"
-                asp-fallback-test="window.jQuery"
-                crossorigin="anonymous"
-                integrity="sha384-K+ctZQ+LL8q6tP7I94W+qzQsfRV2a+AfHIi9k8z8l9ggpc8X+Ytst4yBo/hH+8Fk">
+        <script type="text/javascript" src="~/athena.js"></script>
+
+        @RenderSection("Scripts", required: false)
+
+        <script>
```
Comment (nlowe) [inline src/Athena/Views/Shared/_Layout.cshtml]: Pull these scripts to their own files to be included by the bundler. The only reason we should have inline scripts in views is if it needs dynamic data from the model.

File: src/Athena/Views/Home/Add_Classes.cshtml [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+<div id="first">
+    <h1 id="specific" class="btn" style="width:30em;background-color:navy">Degree Specific Courses</h1>
```
Comment (nlowe) [inline src/Athena/Views/Home/Add_Classes.cshtml]: Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.

File: src/Athena/Views/Home/Add_Classes.cshtml [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+    <div>
+        <p>First Section Content</p>
+    </div>
+    <br />
+    <br />
+    <h1 class="btn" style="width:30em;background-color:navy">All Courses</h1>
```
Comment (nlowe) [inline src/Athena/Views/Home/Add_Classes.cshtml]: Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.

File: src/Athena/Views/Home/Add_Classes.cshtml [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+    <div>
+        <p>Second Section Content</p>
+    </div>
+</div>
+<br />
+<a href="Main_Page" style="width:30em;color:white;background-color:navy" class="btn">Finish</a>
```
Comment (nlowe) [inline src/Athena/Views/Home/Add_Classes.cshtml]: Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.

File: src/Athena/Views/Home/Login_Page.cshtml [hunk 14]
```diff
@@ -0,0 +1,17 @@
...
+
+<div class="input-field col s6">
+    <input id="User_Name" type="text" class="validate">
+    <label for="User_Name">User Name</label>
+</div>
+<div class="input-field col s6">
+    <input id="Password" type="text" class="validate">
+    <label for="Password">Password</label>
+</div>
+<a href="Main_Page" style="color:white" class="btn">Login</a>
```
Comment (nlowe) [inline src/Athena/Views/Home/Login_Page.cshtml]: Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.

File: src/Athena/Views/Home/Login_Page.cshtml [hunk 15]
```diff
@@ -0,0 +1,17 @@
...
+<a href="Home" style="color:white" class="btn">Cancel</a>
```
Comment (nlowe) [inline src/Athena/Views/Home/Login_Page.cshtml]: Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.

File: src/Athena/Views/Home/Main_Page.cshtml [hunk 16]
```diff
@@ -0,0 +1,30 @@
...
+    <div>
+        <h5>ETC:</h5>??/??/????
+    </div>
+
+<br />
+
+<h4>Class Schedule:</h4>
+<div id='calendar'></div>
+
+<a href="Schedule_Page" style="color:white" class="btn">New Schedule</a>
```
Comment (nlowe) [inline src/Athena/Views/Home/Main_Page.cshtml]: Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.

File: src/Athena/Views/Home/Main_Page.cshtml [hunk 17]
```diff
@@ -0,0 +1,30 @@
...
+
+<div class="btn">
+    print
+</div>
+
+<a href="Add_Classes" style="color:white" class="btn">Complete Classes</a>
```
Comment (nlowe) [inline src/Athena/Views/Home/Main_Page.cshtml]: Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.

File: src/Athena/Views/Home/NewAccountPage.cshtml [hunk 18]
```diff
@@ -0,0 +1,22 @@
...
+    <!-- Dropdown Trigger -->
+    <a class='dropdown-button btn' href='#' data-activates='dropdown1'>Select Your University</a>
+
+    <!-- Dropdown Structure -->
+    <ul id='dropdown1' class='dropdown-content'>
+        <li><a href="#!">University one</a></li>
+        <li><a href="#!">University two</a></li>
+    </ul>
+
+    <a href="Main_Page" style="color:white" class="btn indigo darken-4">Create Account</a>
```
Comment (nlowe) [inline src/Athena/Views/Home/NewAccountPage.cshtml]: Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.

File: src/Athena/Views/Home/NewAccountPage.cshtml [hunk 19]
```diff
@@ -0,0 +1,22 @@
...
+    <a href="Home" style="color:white" class="btn">Cancel</a>
```
Comment (nlowe) [inline src/Athena/Views/Home/NewAccountPage.cshtml]: Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.

File: src/Athena/Views/Home/Schedule_Page.cshtml [hunk 20]
```diff
@@ -0,0 +1,32 @@
...
+
+
+
+<div id="name"><ul></ul></div>
+
+
+
+
+<div id="first">
+    <h1 id="specific" class="btn" style="width:30em;background-color:navy">Degree Specific Courses</h1>
```
Comment (nlowe) [inline src/Athena/Views/Home/Schedule_Page.cshtml]: Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.

File: src/Athena/Views/Home/Schedule_Page.cshtml [hunk 21]
```diff
@@ -0,0 +1,32 @@
...
+    <div>
+        <p>First Section Content</p>
+        <input id="enter" type="text">
+        <input type="button" value="Enter name" onclick="displayUserName()">
+    </div>
+    <br />
+    <br />
+    <h1 class="btn" style="width:30em;background-color:navy">All Courses</h1>
```
Comment (nlowe) [inline src/Athena/Views/Home/Schedule_Page.cshtml]: Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.

File: src/Athena/Views/Home/Schedule_Page.cshtml [hunk 22]
```diff
@@ -0,0 +1,32 @@
...
+    <div>
+        <p>Second Section Content</p>
+    </div>
+</div>
+<br />
+<a href="Main_Page" style="width:30em;color:white;background-color:navy" class="btn">Submit</a>
```
Comment (nlowe) [inline src/Athena/Views/Home/Schedule_Page.cshtml]: Use the [material color palette](http://materializecss.com/color.html) for color. This will also help eliminate inline styles.

File: src/Athena/Views/Shared/_Layout.cshtml [hunk 23]
```diff
@@ -2,70 +2,111 @@
...
+                <a href="#" data-activates="mobile-demo" class="right button-collapse"><i class="material-icons">menu</i></a>
+                <ul class="right hide-on-med-and-down">
+                    <li><a href="NewAccountPage">New Account<i class="material-icons right">person_add</i></a></li>
+                    <li><a href="Login_Page">Login<i class="material-icons right">person_outline</i></a></li>
+                </ul>
+                <ul class="side-nav" id="mobile-demo">
+                    <li><a href="NewAccountPage">New Account</a></li>
+                    <li><a href="Login_Page">Login</a></li>
                 </ul>
+                <h2 class="brand-logo no-pad-top" style="margin-top:12px;font-size:3em;font-weight:bold;font-family:'Palatino Linotype'">Athena</h2>
```
Comment (nlowe) [inline src/Athena/Views/Shared/_Layout.cshtml]: Move this inline style to a stylesheet.

File: src/Athena/Views/Home/Add_Classes.cshtml (hunk 11 above)
Comment (nlowe) [inline src/Athena/Views/Home/Add_Classes.cshtml]: Use the [grid system](http://materializecss.com/grid.html) for size and layout control instead of manually specifying position or size.

File: src/Athena/Views/Home/Add_Classes.cshtml (hunk 12 above)
Comment (nlowe) [inline src/Athena/Views/Home/Add_Classes.cshtml]: Use the [grid system](http://materializecss.com/grid.html) for size and layout control instead of manually specifying position or size.

File: src/Athena/Views/Home/Add_Classes.cshtml (hunk 13 above)
Comment (nlowe) [inline src/Athena/Views/Home/Add_Classes.cshtml]: Use the [grid system](http://materializecss.com/grid.html) for size and layout control instead of manually specifying position or size.

File: src/Athena/Views/Home/Main_Page.cshtml [hunk 24]
```diff
@@ -0,0 +1,30 @@
...
+}
+<br /><br /><br /><br /><br />
+
+<img src="~/images/UT-color-vert-trans.gif" class="responsive-img left" style="height:10em; max-width:100%" />
+<br /><br /><br /><br /><br /><br /><br /><br /><br /><br />
+
+
+    <h4>Degree Progress:</h4>
+    <br />
+    <div id="progress-bar" class="all-rounded" style="height:2em;width:10em">
```
Comment (nlowe) [inline src/Athena/Views/Home/Main_Page.cshtml]: Use the [grid system](http://materializecss.com/grid.html) for size and layout control instead of manually specifying position or size.

File: src/Athena/Views/Home/Schedule_Page.cshtml (hunk 20 above)
Comment (nlowe) [inline src/Athena/Views/Home/Schedule_Page.cshtml]: Use the [grid system](http://materializecss.com/grid.html) for size and layout control instead of manually specifying position or size.

File: src/Athena/Views/Home/Schedule_Page.cshtml (hunk 21 above)
Comment (nlowe) [inline src/Athena/Views/Home/Schedule_Page.cshtml]: Use the [grid system](http://materializecss.com/grid.html) for size and layout control instead of manually specifying position or size.

File: src/Athena/Views/Home/Schedule_Page.cshtml (hunk 22 above)
Comment (nlowe) [inline src/Athena/Views/Home/Schedule_Page.cshtml]: Use the [grid system](http://materializecss.com/grid.html) for size and layout control instead of manually specifying position or size.

File: src/Athena/Views/Shared/_Layout.cshtml [hunk 25]
```diff
@@ -2,70 +2,111 @@
...
-            </div>
-            <div class="navbar-collapse collapse">
-                <ul class="nav navbar-nav">
-                    <li><a asp-area="" asp-controller="Home" asp-action="Index">Home</a></li>
-                    <li><a asp-area="" asp-controller="Home" asp-action="About">About</a></li>
-                    <li><a asp-area="" asp-controller="Home" asp-action="Contact">Contact</a></li>
+    <div class="navbar-fixed indigo darken-4">
+        <nav>
+            <div class="nav-wrapper indigo darken-4">
+                <img src="~/images/owl icon.svg" class="left responsive-img" style="max-height:96px; max-width:96px; padding-left: 3em" />
```
Comment (nlowe) [inline src/Athena/Views/Shared/_Layout.cshtml]: Use the [grid system](http://materializecss.com/grid.html) for size and layout control instead of manually specifying position or size.

**Comment by nlowe** [comment]
I will fix the quoting issues from codacy before I merge.
